download_oscar/dod.py

```diff
@@ -161,8 +163,15 @@ def download_checksums(s: sessions.Session, checksum_url: str, headers) -> dict:
         lines = buffer.readlines()
         dictionary = {}
         for line in lines:
-            (key, val) = line.split()
-            dictionary[key] = val
+
+            (hash, filename) = line.split()
```

Comment:
Specifying key and value explicitly naming the true nature of them is very good 👍